{{ cookiecutter.package_name|slugify }}/pyproject.toml

```diff
@@ -268,7 +268,6 @@ testpaths = "tests"
   cmd = """
     streamlit run
       --browser.gatherUsageStats false
-      --global.sharingMode off
```

Comment:
On closer inspection I do agree it is rather opinionated and would be OK to remove it. For my information, why do you prefer to remove this line?

Reply:
It's not recognized by streamlit anymore in the latest poetry compatible version of the project.
We have `streamlit = "^1.3.0"` in `.toml`, and streamlit 1.4 and 1.5, which are now available, and which are both compatible with the previous statement, do not have that flag anymore.